settings.js

```diff
@@ -9,31 +9,41 @@ const mkdirp = require('mkdirp');
 
 var {remote} = require('electron');
 
-
 module.exports.dataFolder = path.join(remote.app.getPath("appData"), "Revolutionary-Games",
                                       "Launcher");
 
 module.exports.tmpDLFolder = path.join(remote.app.getPath("temp"),
                                        "Revolutionary-Games-Launcher");
 
-module.exports.installPath = path.join(module.exports.dataFolder, "Installed");
-
 module.exports.locallyCachedDLFile = path.join(module.exports.dataFolder,
                                                "saved_version_db_v2.json");
 
+module.exports.defaultInstallPath = path.join(module.exports.dataFolder, "Installed");
+
+module.exports.setInstallPath = (directory) => {
```

Comment:
I still think this property should be just added to the main settings object, and have no separate get or set functions.

Reply:
Nevermind, I got it.